packages/big-design/src/utils/messagingHelpers.tsx

```diff
@@ -17,8 +17,6 @@ export interface SharedMessagingProps extends HTMLAttributes<HTMLDivElement> {
 
 export type MessagingType = 'success' | 'error' | 'warning' | 'info';
 
-export type MessagingVariant = 'alert' | 'message' | 'inline';
```

Comment:
Saw this when looking through types. This is unused and was residual from previous alerts/messages implementation.